ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/video.js

```diff
@@ -0,0 +1,61 @@
+// LOGIC: Extract the & UUID of the youtube video and render it in iframe
+const videoCss = () => {
+	return [
+		{
+			inline: true,
+			mime: 'text/css',
+			text: `
+                .video-container {
+                    position: relative;
+                    width: 100%;
+                    height: 0;
+                    padding-bottom: 56.25%;
```

Comment:
What do we need this padding for?

Reply:
It will result in a `<div>` with height equal to 56.25% of the width of its container (a 16:9 aspect ratio).

To know what padding-bottom value to use for each aspect ratio we can divide the height by the width.

16:9 aspect ratio.
9 / 16 = .5625
We need a percentage, so we multiply by 100
.5625 * 100 = 56.25